buildtools/fslex/Lexing.fs

```diff
@@ -1,423 +1,456 @@
 // (c) Microsoft Corporation 2005-2009.
+module FSharp.Text.Lexing
 
 #nowarn "47" // recursive initialization of LexBuffer
 
+open System.Collections.Generic
 
-#if INTERNALIZED_FSLEXYACC_RUNTIME
```

Comment:
It looks like we expect `INTERNALIZED_FSLEXYACC_RUNTIME` to be an option over the source, given we define
```
    <DefineConstants>INTERNALIZED_FSLEXYACC_RUNTIME;$(DefineConstants)</DefineConstants>
```

Reply:
However we only use this in fslex and fsyacc projects.  Maybe it doesn't matter

Reply:
It was factored out in https://github.com/fsprojects/FsLexYacc/pull/144